integr8ly/integreatly-operator
# [MGDAPI-3628] upgrade cloud-resource-operator to v0.39.0

# Issue link
https://issues.redhat.com/browse/MGDAPI-3628

# What
PR that contains the code changes needed to upgrade `cloud-resource-operator` from v0.38.0 to v0.39.0.
Waiting on https://github.com/integr8ly/cloud-resource-operator/pull/413 to be merged, then we can run the [cro release pipeline](https://master-jenkins-csb-intly.apps.ocp-c1.prod.psi.redhat.com/job/Delorean/job/release/job/cro/) to generate the required images/index.
Once that's done, we can run `make vendor/fix`, push the changes to this branch and remove the WIP status from the PR. Finally, the code can be reviewed and PR merged.

# Verification steps
- Install RHOAM on a ccs cluster
- Confirm that a new version of CRO is present and equals to `0.39.0`
- Confirm that Postgres, Redis and BlobStorage resources are provisioned correctly
